eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -424,6 +425,9 @@ object Helpers {
           Some(CurrentRemoteClose(closing.commitments.remoteCommit, closing.remoteCommitPublished.get))
         case _ if closing.nextRemoteCommitPublished.exists(isRemoteCommitConfirmed) =>
           Some(NextRemoteClose(closing.commitments.remoteNextCommitInfo.left.get.nextRemoteCommit, closing.nextRemoteCommitPublished.get))
+        case _ if closing.customRemoteCommitPublished.values.exists(isRemoteCommitConfirmed) =>
+          val (customRemoteCommit, customRemoteCommitPublished) = closing.customRemoteCommitPublished.find(kv => isRemoteCommitConfirmed(kv._2)).get
+          Some(CustomRemoteClose(customRemoteCommit, customRemoteCommitPublished))
```

Comment:
Just to make sure I understand: we will never run into that code, right?
Since the codecs return an empty map on `INPUT_RESTORED`?

Reply:
We do call this method in the `CLOSING` state. However the part in the `INPUT_RESTORED` handler could indeed be removed but I put it there for consistency with the other closing types.